src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -189,6 +189,12 @@ Shows information about the Submariner endpoint connections with other clusters.
 
 Shows information about the Submariner endpoints in the cluster.
 
+### `show brokers`
+
+`subctl show brokers [flags]`
```

Comment:
Are there any flags?

Reply:
yes, there is a `global` `kubeconfig` flag.

```
$ subctl show brokers --help
This command shows information about the Broker in the cluster

Usage:
  subctl show brokers [flags]

Flags:
  -h, --help   help for brokers

Global Flags:
      --kubeconfig string   absolute path(s) to the kubeconfig file(s)
```

Reply:
Ok but you don't have a section that describes them like, eg, L204 below.

Reply:
L204 describes the "global" flag for "all" of `show` commands. Its a common description. Even other `show` subcommands don't have flag description.